### PR TITLE
Add rubocop and travis configuration files

### PR DESCRIPTION
**Pull request by drake1588**

### CODE REVIEW

File: app/channels/application_cable/channel.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (hdf1996) [inline app/channels/application_cable/channel.rb]: In general we add an ignore for this rule, feel free to add it

File: Gemfile [hunk 2]
```diff
@@ -39,6 +39,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  # Rubocop -> Lintern
+  gem 'rubocop', require: false
```
Comment (hdf1996) [inline Gemfile]: In general, we prefer to be explicit about the version of the gems, you can use the one that's on the gemfile.lock as a reference, in this case, you can change this line to:
```ruby
 gem 'rubocop', require: false, '~> 0.55.0'
```

Comment (hdf1996) [reply Gemfile]: If you want to have a better understanding of versioning at gemfile, take a look at https://www.endpoint.com/blog/2013/11/07/specify-versions-for-your-dependencies

**Review by hdf1996 — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (drake1588) [reply Gemfile]: thanks, i'll modify these files and commit

**Review by hdf1996 — APPROVED** [review]

**Pull request merged**